src/pages/trader-tools/_style.js

```diff
@@ -0,0 +1,329 @@
+import styled, { css } from 'styled-components'
+import { Form, Field } from 'formik'
+import { Flex } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import { Button, LinkButton } from 'components/form'
+import device from 'themes/device'
+import FormikSymbolDropdown from 'components/formikComponents/symbolDropdown'
+
+export const StyledText = styled(Text)`
+    font-size: var(--text-size-s);
+
+    @media ${device.mobileL} {
+        font-size: 14px;
+    }
+`
+
+export const AccountTypeButtonWrapper = styled(Flex)`
+    justify-content: space-between;
+`
+
+export const AccountTypeTabItem = styled.div`
+    height: 72px;
+    width: 207px;
```

Comment:
you can make width 20rem, so we dont need to put media for laptop, mobile and tablet, since it will be automatically resized